bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -524,9 +524,9 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- The default filenames are given relative to the root directory
      for the CLM2 data in the CESM distribution -->
 <!-- Plant function types (relative to {csmdata}) -->
-<paramfile phys="clm6_0" >lnd/clm2/paramdata/ctsm60_params.c241119.nc</paramfile>
-<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params.c241119.nc</paramfile>
-<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params.c241119.nc</paramfile>
+<paramfile phys="clm6_0" >lnd/clm2/paramdata/ctsm60_params_nfix.c241119.nc</paramfile>
+<paramfile phys="clm5_0" >lnd/clm2/paramdata/clm50_params_nfix.c241119.nc</paramfile>
+<paramfile phys="clm4_5" >lnd/clm2/paramdata/clm45_params_nfix.c241119.nc</paramfile>
```

Comment:
@wwieder please review this change.

Reply:
do we want to add nfix to the title of the parameter file? This seems unusual to me?  Also, do we typically have the same default parameter list for each physics version?  This is naive on my part.

Reply:
I agree with @wwieder I'd nix nfix in the name of the file. We make so many changes to paramfiles you can't keep adding the list of new things to the filenames.

Reply:
> Also, do we typically have the same default parameter list for each physics version? This is naive on my part.

The values of the parameters are different for each physics version. Hence, we need to have a different parameter file for each physics version. In CESM3.0 we are likely to have two parameter files for clm6_0, one for coupled and one for CRUJRA.

Does that answer your question?

Reply:
> do we want to add nfix to the title of the parameter file? This seems unusual to me? Also, do we typically have the same default parameter list for each physics version? This is naive on my part.

We discussed this and decided the files are fine, in part because we will not be using them for very long.